consumer.go

```diff
@@ -82,12 +82,15 @@ func (c *Consumer) Poll(timeoutMS int) (*rdkafka.Message, error) {
 	switch e := ev.(type) {
 	case *rdkafka.Message:
 		return e, nil
+	case rdkafka.OffsetsCommitted:
+		c.log.Printf("Committed offsets: %v", e.Offsets)
```

Comment:
OffsetsCommitted may carry an error

https://github.com/confluentinc/confluent-kafka-go/blob/1112e2c08a15bce669a99cc35284a690fd2086b8/kafka/event.go#L292-L296

So maybe just print the whole object which will include printing the error?

https://github.com/confluentinc/confluent-kafka-go/blob/1112e2c08a15bce669a99cc35284a690fd2086b8/kafka/event.go#L132-L134

Reply:
Nice catch! Updated.